fastlane/lanes/build.rb

```diff
@@ -532,39 +532,31 @@ def upload_build_to_app_center(
   end
 
   def upload_gutenberg_sourcemaps(sentry_project_slug:, release_version:, build_version:, app_identifier:)
-    # The bundle and source map files are the same for all architectures.
-    gutenberg_bundle = File.join(PROJECT_ROOT_FOLDER, 'Pods/Gutenberg/Frameworks/Gutenberg.xcframework/ios-arm64/Gutenberg.framework')
-
-    Dir.mktmpdir do |sourcemaps_folder|
```

Comment:
We no longer need to create a temporary path because the files are already located in a separate folder.